packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -30,6 +42,8 @@ const FilterDropdown = ({
                 suffix_icon='IcFilter'
                 should_show_empty_option={false}
                 onChange={onChange}
+                label={label}
```

Comment:
Do we need these 2? because we don't have it in the master :thinking:

Reply:
These 2 are used in select native component.That is why i used